yarn.config.cjs

```diff
@@ -95,20 +95,6 @@ module.exports = defineConfig({
           // All non-root packages must set up ESM- and CommonJS-compatible
           // exports correctly.
           expectCorrectWorkspaceExports(workspace);
-          // All non-root packages must have the same "build" script.
-          if (workspace.ident !== '@metamask/design-tokens') {
-            expectWorkspaceField(
-              workspace,
-              'scripts.build',
-              'ts-bridge --project tsconfig.build.json --verbose --clean --no-references',
-            );
-          } else if (workspace.ident === '@metamask/design-tokens') {
-            expectWorkspaceField(
-              workspace,
-              'scripts.build',
-              'ts-bridge --project tsconfig.build.json --verbose --clean --no-references && yarn build:css',
-            );
-          }
```

Comment:
Removing standardized build script checks from constraints because:
1. Our packages have diverse build requirements:
- Design tokens needs CSS generation
- React components need SVGR icon generation
- React Native has its own build process
2. Forcing a single build command pattern (ts-bridge) across all packages doesn't make sense when build steps vary significantly
3. Each package should be able to define its own build process based on its specific needs

This change allows packages to maintain build scripts that are appropriate for their individual requirements rather than forcing a one-size-fits-all approach.